app/views/application/success.scala.html

```diff
@@ -9,13 +9,14 @@ <h1 class="heading-large no-btm-margin"><i class="fa fa-check"></i>Application s
         <p><i class="fa fa-envelope-o"></i>We've sent you an email confirming your
             application</p>
     </div>
-    <h2 class="heading-large">What happens next?</h2>
-    <div class="text">
-        <p>The next step is to complete the online tests. You'll be sent an email once these are ready for you to start and you'll have 7 days to complete them.</p>
-        <p>If you can't see the email in your inbox, check your spam folder.</p>
-        <p><a href="@feedbackUrl" target="_blank" rel="external">Give feedback?</a> (30 second survey)</p>
-        <p>Finished with the service? <a href="@routes.SignInController.signOut">Sign out</a>.</p>
-    </div>
-
+    <section class="section-border">
+        <div class="text">
+            <h2 class="heading-large">What happens next?</h2>
+            <p>The next step is to complete the online exercises.</p>
+            <a href="@routes.HomeController.present" class="button">Return to home</a>
```

Comment:
Sorry about this, but the pedant strikes back - routes.HomeController.present should have parenthesis,  as per our team chat the other day.